packages/components/src/components/List/OrderedList.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ReactElement } from "react";
+import { Box } from "../../primitives/Box";
+import { ListItemProps } from "./ListItem";
+
+export interface OrderedListProps
+  extends Omit<React.HTMLAttributes<HTMLOListElement>, "color"> {
+  /** The list items */
+  children: ReactElement<ListItemProps> | ReactElement<ListItemProps>[];
```

Comment:
Why doesn't this component have the colors with that two tokens as type prop?

Reply:
I just removed the color https://github.com/Localitos/pluto/pull/147#discussion_r1005886838